samples/features/security/ledger/azure-sql-database/setup/PopulateDatabase.sql

```diff
@@ -0,0 +1,374 @@
+DELETE FROM [dbo].[Employees];
+DELETE FROM [dbo].[AspNetRoles];
+DELETE FROM [dbo].[AspNetUsers];
+DELETE FROM [dbo].[AspNetUserRoles];
+GO
+ 
+DECLARE @SSN1 char(11) = '795-73-9833'; DECLARE @Salary1 Money = 61692.00; INSERT INTO [dbo].[Employees] ([SSN], [FirstName], [LastName], [Salary]) VALUES (@SSN1, 'Catherine', 'Abel', @Salary1);
```

Comment:
You could have INSERT using Table Value Constructor. It is single statement with 100s of inserts.

INSERT INTO Employees (...)
VALUES( 'ssn1', 'C', 'Ab'. 19284.00),
('ssn2'. 'B', 'B2', 1294.),
('ssn3'. 'C', 'C2', 498842.);

Reply:
Agreed, I will keep it in mind for a next release